themes/bootstrap3/templates/record/hold.phtml

```diff
@@ -31,7 +31,7 @@
     </div>
   <?php endif; ?>
 
-  <?php if (in_array("requiredByDate", $this->extraHoldFields)): ?>
+  <?php if (in_array("requiredByDate", $this->extraHoldFields) || in_array("requiredByDateOptional", $this->extraHoldFields)): ?>
```

Comment:
Do we need to display a different label, or some other visual indicator, to differentiate between optional and non-optional versions of the behavior?

Reply:
Added!